tests/test_asset_exchange.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+import logging
+from unittest.mock import MagicMock
+from threading import Thread
+from wsgiref.simple_server import WSGIRequestHandler
+
+from falcon import App
+import pytest
+
+from proof_of_concept.components.asset_store import AssetStore
+from proof_of_concept.definitions.assets import DataAsset
+from proof_of_concept.definitions.identifier import Identifier
+from proof_of_concept.definitions.registry import RegisteredObject
+from proof_of_concept.replication import ReplicaUpdate
+from proof_of_concept.rest.client import SiteRestClient
+from proof_of_concept.rest.ddm_site import (
+        AssetImageAccessHandler, ThreadingWSGIServer)
+
+
+@pytest.fixture
+def asset_id():
+    return Identifier('asset:ns:test_asset:ns:site')
+
+
+@pytest.fixture
+def asset_store(temp_path, test_image_file, asset_id):
+    asset_store = AssetStore(MagicMock(), temp_path)
+    asset = DataAsset(asset_id, None, str(test_image_file))
+    asset_store.store(asset)
+    return asset_store
+
+
+@pytest.fixture
+def image_server(asset_store):
```

Comment:
Why do we need a separate image server here? The AssetImageAccessHandler is part of the ddm site server right? Can we not use a ddm site server?

Reply:
We could, but the DDM site server also serves a bunch of other stuff, so I'd have to mock everything else in the site. This seemed like less work.

Reply:
Cool